### PR TITLE
chore(C1.1): fix spelling and phrasing

### PR DESCRIPTION
**Pull request by nyarthan**

### CODE REVIEW

File: src/chapter1/01-hello-world.md [hunk 1]
```diff
@@ -182,9 +182,9 @@ console.log('Hello, world!');
...
+> There is a lot of discussion on how much you should comment your programs. We will return to this when discussing functions. However one rule is that if your code is so terrible that it requires *extensive commentary* to explain its behavior or purpose, you should fix the code. Just like a work of art, your code stands on it's own merits. Imagine *commenting* a work of art (oh, wait)... However it is better to have terrible code and comments than have terrible code and no comments.
 
-We will also adopt the convention that if a comment is next to a line with a `console.log` call that comment shows the output of that call. For example:
+We will also adopt the convention that if a comment is next to a line with a `console.log` statement, that comment shows the output that would be logged to the console if the code was run. For example:
```
Comment (Jannis-Morgenstern) [inline src/chapter1/01-hello-world.md:187]: they don't know what a "call" is yet

File: src/chapter1/01-hello-world.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ This is how the browser console will approximately look like on Firefox:
 
 ![](images/browser-console.png)
 
-> Note that if you see a bunch of scary error or warning messages upon opening the console, *don't panic* (this is also good life advice in general). Most of these will probably come from from various extensions you might have installed. Simply delete the messages by clicking the trash can icon, as we don't care about them.
+> Note that if you see a bunch of scary error or warning messages upon opening the console, *don't panic* (this is also good life advice in general). Most of these will probably come from various extensions you might have installed or the web page you are currently viewing. Simply delete the messages by clicking the icon in the top left of the console (trash can in firefox - might vary in different browsers), as we don't care about them.
```
Comment (uhasker) [inline src/chapter1/01-hello-world.md:24]: Firefox should probably be capitalized

File: src/chapter1/01-hello-world.md [hunk 3]
```diff
@@ -182,9 +182,9 @@ console.log('Hello, world!');
 
 We will heavily utilize comments throughout this book inside the code blocks to highlight important ideas.
 
-> There is a lot of discussion on how much you should comment your programs. We will return to this when discussing functions. However one rule is that if your code is so terrible that it requires *extensive commentary* to explain its purpose, you should fix the code. Just like a work of art, your code stand on it's own merits. Imagine *commenting* a work of art (oh, wait)... However it is better to have terrible code and comments than have terrible code and no comments.
+> There is a lot of discussion on how much you should comment your programs. We will return to this when discussing functions. However one rule is that if your code is so terrible that it requires *extensive commentary* to explain its behavior or purpose, you should fix the code. Just like a work of art, your code stands on it's own merits. Imagine *commenting* a work of art (oh, wait)... However it is better to have terrible code and comments than have terrible code and no comments.
```
Comment (uhasker) [inline src/chapter1/01-hello-world.md:186]: stands -> should stand

File: src/chapter1/01-hello-world.md (hunk 2 above)
Comment (Jannis-Morgenstern) [reply src/chapter1/01-hello-world.md:24]: fixed

File: src/chapter1/01-hello-world.md (hunk 3 above)
Comment (Jannis-Morgenstern) [reply src/chapter1/01-hello-world.md:186]: fixed

**Review by uhasker — APPROVED** [review]

**Pull request merged**